packages/function-entry/src/hints-popover.jsx

```diff
@@ -0,0 +1,142 @@
+import { withStyles } from 'material-ui/styles';
+import React from 'react';
+import PropTypes from 'prop-types';
+import Popover from 'material-ui/Popover';
+import Typography from 'material-ui/Typography';
+
+const styles = theme => ({
+  fraction: {
+    display: 'inline-block',
+    position: 'relative',
+    'vertical-align': 'middle',
```

Comment:
can you do `verticalAlign: 'middle'` instead, and below too - use camelCase instead of stringy kebab-case